css-ecommerce-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
+    </header>
+  </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-2">
+        <img src="images/dress1.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/ysl-bag.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/key.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p>Free people <span> |Size 6</span></p>
+      </div>
```

Comment:
The capitalization is off here:

---

![image](https://user-images.githubusercontent.com/16143771/152213616-10de94fb-387d-4359-90f6-6fe10ec48419.png)